doc/source/conf.py

```diff
@@ -69,11 +70,12 @@
 # Favicon
 html_favicon = ansys_favicon
 
-extensions = [
-    "sphinx_design",
-    "sphinx_copybutton",
-    "sphinxcontrib.mermaid",
-]
+extensions = ["sphinx_design", "sphinx_copybutton", "sphinxcontrib.mermaid", "sphinx_jinja"]
+
+metadata_path = Path(__file__).parent.parent.parent / "projects-metadata.yaml"
```

Comment:
Maybe we could rename the file just to `projects.yml`.

Reply:
Agreed